### ISSUE TITLE
focal-270-2.7.1 install error

### ISSUE THREAD

**Issue by mauser8004** [bigbluebutton/bigbluebutton#18927]
Today, the error occurred in the case of two servers, and in the first case it was restored only after 10-15 minutes

universe' distribution component is already enabled for all sources.
Warning: apt-key output should not be parsed (stdout is not a terminal)
Cannot add PPA: 'ppa:~martin-uni-mainz/ubuntu/coturn'.
The user named '~martin-uni-mainz' has no PPA named 'ubuntu/coturn'
Please choose from the following available PPAs:
 * 'coturn':  Coturn
Warning: apt-key output should not be parsed (stdout is not a terminal)
Cannot add PPA: 'ppa:~martin-uni-mainz/ubuntu/coturn'.
The user named '~martin-uni-mainz' has no PPA named 'ubuntu/coturn'
Please choose from the following available PPAs:
 * 'coturn':  Coturn
Warning: apt-key output should not be parsed (stdout is not a terminal)
bbb-install: Unable to setup PPA for ppa:martin-uni-mainz/coturn

**Comment by mauser8004** [comment]
After a rerun this

'universe' distribution component is already enabled for all sources.
Warning: apt-key output should not be parsed (stdout is not a terminal)
Traceback (most recent call last):
  File "/usr/bin/add-apt-repository", line 137, in <module>
    shortcut = shortcut_handler(line)
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py", line 885, in shortcut_handler
    ret = factory(shortcut)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 469, in shortcut_handler
    return PPAShortcutHandler(shortcut)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 426, in __init__
    info = get_ppa_info(self.shortcut)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 392, in get_ppa_info
    _get_suggested_ppa_message(user, ppa))
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 349, in _get_suggested_ppa_message
    lp_user = get_info_from_lp(LAUNCHPAD_USER_API % user)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 104, in get_info_from_lp
    return get_info_from_https(lp_url, True)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 98, in get_info_from_https
    return json.loads(data)
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Warning: apt-key output should not be parsed (stdout is not a terminal)

**Comment by mauser8004** [comment]
running it for the fourth time, exceptionally good

**Comment by antobinary** [comment]
A detailed answer after some investigations: https://github.com/bigbluebutton/bbb-install/issues/689#issuecomment-1758312139
tl;dr: likely a brief service outage. No evidence of an issue on our side. Please comment again if this happens again (or anything else to add). Thank you for reporting this!
